translate/src/core/editor/components/Editor.test.js

```diff
@@ -26,6 +26,7 @@ const NESTED_SELECTORS_STRING = ftl`
           }
           [select] WOW
       }
+
```

Comment:
Is this intentional? Prettier?

Reply:
It's intentional, and required for the test to pass with the normalisation that's now applied.

Reply:
Oh, that's a message, not code. :)